### ISSUE TITLE
Fails Lighthouse accessibility audit on Labels

### ISSUE THREAD

**Issue by SteveX** [adaptlearning/adapt-contrib-confidenceSlider#60]
Form elements do not have associated labels
Labels ensure that form controls are announced properly by assistive technologies, like screen readers.

Failing Elements
div.component__inner > div.component__widget > div.slider__item > input.slider__item-input
<input class="slider__item-input js-slider-item-input" type="range" role="slider" min="1" max="10" aria-valuenow="1" aria-valuemin="1" aria-valuemax="10" data-rangeslider="true" data-direction="false" value="1" step="1" style="position: absolute; width: 1px; height: 1px; overflow: hidden; opacity: 0;">

**Comment by oliverfoster** [comment]
duplicate of https://github.com/adaptlearning/adapt-contrib-slider/issues/156
